### ISSUE TITLE
npm run example_1 throws error

### ISSUE THREAD

**Issue by filcole** [mdocter/xrm-examples#1]
git clone https://github.com/mdocter/xrm-examples.git
cd xrm-examples/

npm install
npm run example_1

gives me the following errors.

Am I doing something wrong?

```
C:\Users\Phil\Documents\Source\xrm-examples\example_1>npm run example_1

> xrm-examples@1.0.0 example_1 C:\Users\Phil\Documents\Source\xrm-examples
> webpack --config ./example_1/webpack.config.js --display-error-details

Hash: df9d3d8d3c32b6b76585
Version: webpack 1.12.9
Time: 2578ms
                                Asset     Size  Chunks             Chunk Names
    dist/example_1/mdocter.account.js  2.91 kB       0  [emitted]  mdocter.account
    dist/example_1/mdocter.contact.js  2.01 kB       1  [emitted]  mdocter.contact
dist/example_1/mdocter.account.js.map  5.27 kB       0  [emitted]  mdocter.account
dist/example_1/mdocter.contact.js.map  2.04 kB       1  [emitted]  mdocter.contact
   [0] multi mdocter.account 28 bytes {0} [built]
   [0] multi mdocter.contact 28 bytes {1} [built] [1 error]
    + 4 hidden modules

ERROR in ./contact.ts
Module build failed: Error: File C:\Users\Phil\Documents\Source\xrm-examples\example_1\entrypoints\contact.ts was not found in program
    at State.emit (C:\Users\Phil\Documents\Source\xrm-examples\node_modules\awesome-typescript-loader\dist.babel\host.js:254:27)
    at transform (C:\Users\Phil\Documents\Source\xrm-examples\node_modules\awesome-typescript-loader\dist.babel\index.js:168:48)
    at _callee2$ (C:\Users\Phil\Documents\Source\xrm-examples\node_modules\awesome-typescript-loader\dist.babel\index.js:218:42)
    at tryCatch (C:\Users\Phil\Documents\Source\xrm-examples\node_modules\awesome-typescript-loader\node_modules\babel-polyfill\node_modules\babel-regenerator-runtime\runtime.js:61:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\Phil\Documents\Source\xrm-examples\node_modules\awesome-typescript-loader\node_modules\babel-polyfill\node_modules\babel-regenerator-runtime\runtime.js:329:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\Phil\Documents\Source\xrm-examples\node_modules\awesome-typescript-loader\node_modules\babel-polyfill\node_modules\babel-regenerator-runtime\runtime.js:94:21)
    at step (C:\Users\Phil\Documents\Source\xrm-examples\node_modules\awesome-typescript-loader\dist.babel\index.js:30:41)
    at onfulfill (C:\Users\Phil\Documents\Source\xrm-examples\node_modules\awesome-typescript-loader\dist.babel\index.js:17:17)
    at tryCatcher (C:\Users\Phil\Documents\Source\xrm-examples\node_modules\awesome-typescript-loader\node_modules\bluebird\js\main\util.js:26:23)
    at Promise._settlePromiseFromHandler (C:\Users\Phil\Documents\Source\xrm-examples\node_modules\awesome-typescript-loader\node_modules\bluebird\js\main\promise.js:507:31)
    at Promise._settlePromiseAt (C:\Users\Phil\Documents\Source\xrm-examples\node_modules\awesome-typescript-loader\node_modules\bluebird\js\main\promise.js:581:18)
    at Promise._settlePromiseAtPostResolution (C:\Users\Phil\Documents\Source\xrm-examples\node_modules\awesome-typescript-loader\node_modules\bluebird\js\main\promise.js:245:10)
    at Async._drainQueue (C:\Users\Phil\Documents\Source\xrm-examples\node_modules\awesome-typescript-loader\node_modules\bluebird\js\main\async.js:128:12)
    at Async._drainQueues (C:\Users\Phil\Documents\Source\xrm-examples\node_modules\awesome-typescript-loader\node_modules\bluebird\js\main\async.js:133:10)
    at Immediate.Async.drainQueues [as _onImmediate] (C:\Users\Phil\Documents\Source\xrm-examples\node_modules\awesome-typescript-loader\node_modules\bluebird\js\main\async.js:15:14)
    at processImmediate [as _immediateCallback] (timers.js:383:17)
 @ multi mdocter.contact
```

**Comment by mdocter** [comment]
Can't test right now, but it looks like you're running 'npm run example_1' from the example_1 folder. it should also be run from the root folder. Can you give that a try?

**Comment by filcole** [comment]
It doesn't work from there either.  Looking into it I think the problem is
to do with https://github.com/s-panferov/awesome-typescript-loader/issues/9

Are you on Windows?

On 6 December 2015 at 14:22, Maarten Docter notifications@github.com
wrote:

> Can't test right now, but it looks like you're running 'npm run example_1'
> from the example_1 folder. it should also be run from the root folder. Can
> you give that a try?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/mdocter/xrm-examples/issues/1#issuecomment-162320756.

## 

Phil

**Comment by mdocter** [comment]
Hi Phil, 

I'm on Windows. For some reason this issue wasn't occurring in my development folder (said the programmer :-) ). After cloning the repo to a new clean folder I had the same.
I used the 'awesome-typescript-loader' because the TypeScript site referenced it (can't remember where I saw it exactly). Now I switched to [ts-loader](https://github.com/TypeStrong/ts-loader) and that one has more stars on Github and doesn't have this issue. 
So if you do an `git pull origin` (to get the latest commits) and `npm install` (to install the new loader) , it should build fine if you run `npm run example_1` from the root folder. 
Sorry for the inconvenience this has caused you and thank you for reporting it to me.
Let me know if this works, so I can close the issue afterwards.

Cheers,

Maarten
